libs/vm_monitor/src/filecache.rs

```diff
@@ -87,28 +89,37 @@ impl FileCacheConfig {
         // As shown in `calculate_cache_size`, we have two lines resulting from `resource_multiplier` and
         // `spread_factor`, respectively. They are:
```

Comment:
I think you also need to adjust the description of spread_factor in line 53 and above where it says:

`spread_factor` is too large if `(spread_factor + 1) * resource_multiplier >= 1`

Because if we go for a resource_multiplier >=1 that statement is no longer true.

Reply:
Sounds good, will fix them after my vacation :)